platform/ios/src/MGLMapView.mm

```diff
@@ -2512,10 +2512,15 @@ - (void)resetPosition
 {
     MGLLogInfo(@"Resetting the map to the current style’s default viewport.");
     auto camera = self.mbglMap.getStyle().getDefaultCamera();
-    CGFloat pitch = *camera.pitch;
-    CLLocationDirection heading = mbgl::util::wrap(*camera.bearing, 0., 360.);
-    CLLocationDistance altitude = MGLAltitudeForZoomLevel(*camera.zoom, pitch, 0, self.frame.size);
-    self.camera = [MGLMapCamera cameraLookingAtCenterCoordinate:MGLLocationCoordinate2DFromLatLng(*camera.center)
+    
+    double pitch        = camera.pitch ? *camera.pitch : 0.0;
```

Comment:
I take it a `nil` coalescing approach (`?:`) wouldn’t work here?

Reply:
Nope, because `pitch` here is essentially a pointer.

Reply:
We may have a few other cases like this hanging around - but nothing fired, and this is the only use of `getDefaultCamera()`